articles/product-lifecycle/migrations.md

```diff
@@ -34,6 +34,14 @@ We are actively migrating customers to new behaviors for all **Deprecations** li
         <a href="/users/search/v3/migrate-search-v2-v3">User Search v2 to v3 Migration Guide</a><br>
       </td>
     </tr>
+    <tr>
+      <td><a href="/logs/migrate-logs-v2-v3">`context` field for Facebook Connection (3rd party deprecation)</a></td>
+      <td>30 April 2019</td>
+      <td>
+        30 July 2019
+      </td>
+      <td>On April 30th 2019, Facebook deprecated the use of the ‘Social Context’ field for new applications. https://developers.facebook.com/docs/graph-api/changelog/4-30-2019-endpoint-deprecations.</td>
```

Comment:
@aaguiarz We should include a link here to either a community post with more details, or a new (short lived) page at something like https://auth0.com/docs/connections/social/facebook/migrate-context-field